Description: Fix spelling error in binary.
Author: Scarlett Moore <sgclark@kde.org>
Bug: https://github.com/AppImage/libappimage/issues/46
Last-Update: 2019-01-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: libappimage/src/libappimage_shared/elf.c
===================================================================
--- libappimage.orig/src/libappimage_shared/elf.c	2019-01-09 05:42:49.721466452 -0700
+++ libappimage/src/libappimage_shared/elf.c	2019-01-11 09:07:48.259236674 -0700
@@ -136,7 +136,7 @@
 	}
 	if ((ehdr.e_ident[EI_DATA] != ELFDATA2LSB) &&
 		(ehdr.e_ident[EI_DATA] != ELFDATA2MSB)) {
-		fprintf(stderr, "Unkown ELF data order %u\n",
+		fprintf(stderr, "Unknown ELF data order %u\n",
 			ehdr.e_ident[EI_DATA]);
 		return -1;
 	}
